CPAC/anat_preproc/anat_preproc.py

```diff
@@ -2534,154 +2536,143 @@ def brain_extraction_temp_T2(wf, cfg, strat_pool, pipe_num, opt=None):
+    #                             name=f'fs_aseg_to_native_{pipe_num}')
+    # fs_aseg_to_native.inputs.reg_header = True
+    # fs_aseg_to_native.inputs.interp = 'nearest'
     
-    wf.connect(node, out, fs_aseg_to_native, 'subjects_dir')
+    # wf.connect(node, out, fs_aseg_to_native, 'subjects_dir')
 
-    node, out = strat_pool.get_data('subcortical-seg')
-    wf.connect(node, out, fs_aseg_to_native, 'source_file')
+    # node, out = strat_pool.get_data('pipeline-fs_subcortical-seg')
+    # wf.connect(node, out, fs_aseg_to_native, 'source_file')
     
-    node, out = strat_pool.get_data('raw-average')
-    wf.connect(node, out, fs_aseg_to_native, 'target_file')
-
-    # convert registered FS segmentations from .mgz to .nii.gz
-    fs_aseg_to_nifti = pe.Node(util.Function(input_names=['in_file'],
-                                             output_names=['out_file'],
-                                             function=mri_convert),
-                               name=f'fs_aseg_to_nifti_{pipe_num}')
-    fs_aseg_to_nifti.inputs.args = '-rt nearest'
-
-    wf.connect(fs_aseg_to_native, 'transformed_file',
-               fs_aseg_to_nifti, 'in_file')
-
-    pick_tissue = pe.Node(pick_tissue_from_labels_file_interface(),
-                          name=f'select_fs_tissue_{pipe_num}')
-
-    pick_tissue.inputs.csf_label = cfg['segmentation'][
-        'tissue_segmentation']['FreeSurfer']['CSF_label']
-    pick_tissue.inputs.gm_label = cfg['segmentation'][
-        'tissue_segmentation']['FreeSurfer']['GM_label']
-    pick_tissue.inputs.wm_label = cfg['segmentation'][
-        'tissue_segmentation']['FreeSurfer']['WM_label']
-
-    wf.connect(fs_aseg_to_nifti, 'out_file', pick_tissue, 'multiatlas_Labels')
-
-    erode_tissues = {}
-    if cfg['segmentation']['tissue_segmentation']['FreeSurfer']['erode'] > 0:
-        for tissue in ['csf', 'wm', 'gm']:
-            erode_tissues[tissue] = pe.Node(
-                interface=freesurfer.model.Binarize(),
-                name=f'erode_{tissue}_{pipe_num}')
-            erode_tissues[tissue].inputs.match = [1]
-            erode_tissues[tissue].inputs.erode = cfg['segmentation'][
-                'tissue_segmentation']['FreeSurfer']['erode']
-            wf.connect(pick_tissue, f'{tissue}_mask', erode_tissues[tissue],
-                       'in_file')
-
-    outputs = {
-        'space-T1w_desc-brain_mask': (fill_fs_brain_mask, 'out_file')
-    }
-
-    if erode_tissues:
-        outputs['label-CSF_mask'] = (erode_tissues['csf'], 'binary_file')
-        outputs['label-WM_mask'] = (erode_tissues['wm'], 'binary_file')
-        outputs['label-GM_mask'] = (erode_tissues['gm'], 'binary_file')
-    else:
-        outputs['label-CSF_mask'] = (pick_tissue, 'csf_mask')
-        outputs['label-WM_mask'] = (pick_tissue, 'wm_mask')
-        outputs['label-GM_mask'] = (pick_tissue, 'gm_mask')
-
-    return (wf, outputs)
+    # node, out = strat_pool.get_data('pipeline-fs_raw-average')
```

Comment:
If we're ready to let this code go, I would say remove anything that is commented out. If there's something we're still saving about it, let's take a note and archive it.